webpack.dev-site.config.js

```diff
@@ -0,0 +1,34 @@
+/* eslint-disable import/no-extraneous-dependencies */
+const merge = require('webpack-merge');
+const {
+  TerraDevSite,
+  TerraDevSiteEntrypoints,
+  DirectorySwitcherPlugin,
+  LocalPackageAliasPlugin,
+} = require('terra-dev-site');
+
+const WebpackConfigTerra = require('./packages/webpack-config-terra/webpack.config');
```

Comment:
It's nice to see that the new webpack config works within the mono repo and terra-dev-site can continue to work with it.